habitat/tasks/nav/nav_task.py

```diff
@@ -491,6 +510,29 @@ def reset_metric(self, episode):
             self._map_resolution,
         )
         self._previous_xy_location = (a_y, a_x)
+        self._draw_shortest_path()
+
+    def _draw_shortest_path(self):
```

Comment:
Ideally, this should be independent of the task and a separate utility function which can be used by any class.